lib/gold.ex

```diff
@@ -12,18 +12,18 @@ defmodule Gold do
   @doc """
   Starts GenServer link with Gold server.
   """
-  def start_link(config), do: GenServer.start_link(__MODULE__, config)
+  def start_link(config, opts \\ []), do: GenServer.start_link(__MODULE__, config, opts)
 
   @doc """
   Returns server's total available balance.
   """
   def getbalance(pid) do
     case GenServer.call(pid, :getbalance) do
```

Comment:
All those are extremelly usefull to stop the linter from complaining, :+1: on my side as well.